src/components/FieldBooleanCheckbox/index.tsx

```diff
@@ -55,7 +55,7 @@ export const FieldBooleanCheckbox = (props: FieldBooleanCheckboxProps) => {
       <Checkbox
         id={id}
         size={size}
-        value={value ?? false}
+        value={`${value}`}
```

Comment:
why this change is needed ? You cast your boolean value into a string ?

Reply:
Because Checkbox component value cannot be a boolean, it must be a string